ecom-css/footer.css

```diff
@@ -0,0 +1,36 @@
+
+.footer {
+  background-color: var(--nav-bg-color);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin-top: 5rem;
+}
+
+.footer h2 {
+  font-size: 2rem;
+  color: var(--white);
+  margin: 2rem 0;
+  font-weight: 400;
+}
+
+.footer-icon-container {
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 3rem;
+  margin-bottom: 2rem;
```

Comment:
another way to write the margins is margin: 0 auto 2rem, just a suggestion as sudesh once mentioned about it.

Reply:
yeah thanks for that partha this actually missed my eye